cmd/registry/config.yml

```diff
@@ -4,6 +4,20 @@ log:
   fields:
     service: registry
     environment: development
+  hooks:
+    type: none # set to "email", and allows user to send email 
```

Comment:
Rather than have this sentinel value, let's add an enabled field that can be use to disable a configured hook.